upgrades-6to6.adoc

```diff
@@ -28,3 +28,99 @@ $ docker-compose up -d
 ----
 
 This will take care of new static and the database migrations, if any. This process means a little downtime so choose the better moment to perfom it. After this process the instance will be updated and running.
+
+== Advanced Upgrade Process
```

Comment:
All this additions are out of the scope of the original PR.
These explanations are more a tutorial than a documentation, and currently it's not the content we're adding.

In the near future we'll be rethinking all the documentation/tutorials/resources and we're sure that we'll find a good match for all the needs.

Please, remove this changes from the PR.

Reply:
Done ✅